### PR TITLE
DOC-6581 - New WorkloadStatus event and attributes

### PR DESCRIPTION
**Pull request by barbnewrelic**
Test updates to the data dictionary so Tech Docs team can help me troubleshoot new WorkloadStatus event and its attributes. I don't think this is ready to merge yet because I can't see the new event in my localhost, not sure what I did wrong. Thanks!

### CODE REVIEW

File: src/data-dictionary/events/WorkloadStatus/statusValueCode.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: statusValueCode
+type: attribute
+dataSources:
```
Comment (roadlittledawn) [inline src/data-dictionary/events/WorkloadStatus/statusValueCode.md:4]: This should be `events` and not `dataSources`

**Comment by barbnewrelic** [comment]
@helenapm would you like to look at the Files Changed tab? We're still troubleshooting the Read more formatting so I temporarily removed it but will put it back in.

**Comment by helenapm** [comment]
Hi @barbnewrelic , thanks for the note. Changes look good to me, although Github doesn't make it easy to picture how the document will look in the end :) Thanks!

**Comment by zuluecho9** [comment]
Just a heads up that I removed those links again, sorry to go back and forth. It made me realize we need better standards around adding links, because those things are now in the short description (no long description separation), which means most ways we word link wordings will look bad in the product query UI view of these things. So we need to come up with better standards here.

**Comment by barbnewrelic** [comment]
Thanks for letting me know! This has been a good experience for discovering some things we still need to tweak over time, not just for Workload.

**Review by zuluecho9 — APPROVED** [review]

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**